frontend/src/common/ImageComponent.js

```diff
@@ -0,0 +1,22 @@
+import React, {useEffect, useState} from 'react'
+
+function ImageComponent({imageId}) {
+
+  const [state, setState] = useState({
+    base64String: null
+  })
+
+  useEffect(() => {
+    fetch('/api/image/' + imageId, {method:"GET"})
+      .then((response) => response.json())
+      .then((json) => setState({base64String: json.base64String}))
+  }, [imageId])
+
+  return (
+    <>
+      {state.base64String && <img src={state.base64String} alt="image" style={{width: '100%'}}/>}
+    </>
+  )
+}
+
```

Comment:
Maybe we can add a propType ensuring the `imageId` is required or set a default value for it? Like the placeholder suggestion.

Reply:
A placeholder id is not easy to do. We need to have the default image uploaded to the database at all times and we need to somehow know it's id. Making the `imageId` a required field is a better option.

Reply:
Yes as if I understood correctly default value for base64 would be enough.